vms/event/templates/event/create.html

```diff
@@ -90,74 +90,47 @@
-                        </div>
-                    </div>
-                {% endif %}
-				
-				{% if form.state.errors %}
-                    <div class="form-group has-error">
-                        <label class="col-md-2 control-label">{% trans "State" %}</label>
-                        <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Event State{% endblocktrans %}" type="text" name="state" value="{% if form.state.value %}{{ form.state.value }}{% endif %}">
-                            <p class="help-block">
-                                <strong>
-                                    {% for error in form.state.errors %}
-                                        {{ error }}
-                                    {% endfor %}
-                                </strong>
-                            </p>
-                        </div>
-                    </div>
-                {% else %}
-                    <div class="form-group">
-                        <label class="col-md-2 control-label">{% trans "State" %}</label>
-                        <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Event State{% endblocktrans %}" type="text" name="state" value="{% if form.state.value %}{{ form.state.value }}{% endif %}">
-                        </div>
-                    </div>
-                {% endif %}
-				
-				{% if form.city.errors %}
-                    <div class="form-group has-error">
-                        <label class="col-md-2 control-label">{% trans "City" %}</label>
-                        <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Event City{% endblocktrans %}" type="text" name="city" value="{% if form.city.value %}{{ form.city.value }}{% endif %}">
-                            <p class="help-block">
-                                <strong>
-                                    {% for error in form.city.errors %}
-                                        {{ error }}
-                                    {% endfor %}
-                                </strong>
-                            </p>
-                        </div>
-                    </div>
-                {% else %}
-                    <div class="form-group">
-                        <label class="col-md-2 control-label">{% trans "City" %}</label>
-                        <div class="col-md-8">
-                            <input class="form-control" placeholder="{% blocktrans %}Event City{% endblocktrans %}" type="text" name="city" value="{% if form.city.value %}{{ form.city.value }}{% endif %}">
-                        </div>
-                    </div>
-                {% endif %}
+            <div class="form-group">
```

Comment:
For this page, you haven't implemented "filter based on country and state" functionality.

Reply:
done

Reply:
Did you commit the changes?